src/Generating/ComposableGenerator.cpp

```diff
@@ -438,6 +438,11 @@ void cComposableGenerator::InitFinishGens(cIniFile & a_IniFile)
 				m_FinishGens.push_back(gen);
 			}
 		}
+		else if (NoCaseCompare(finisher, "OverworldClumpFlowers") == 0)
+		{
+			AString flowers = a_IniFile.GetValueSet("Generator", "OverworldClumpFlowers", "Forest+-2+2,ForestHills+-3+2,FlowerForest; yellowflower,redflower,lilac,rosebush | plains+-2+1,SunflowerPlains; yellowflower,redflower,azurebluet,oxeyedaisy | SunflowerPlains+1+2; sunflower | FlowerForest+2+5; allium,redtulip,orangetulip,whitetulip,pinktulip,oxeyedaisy | Swampland,SwamplandM; brownmushroom,redmushroom,blueorchid");
```

Comment:
@madmaxoft again, the style checker really shouldn't look at comments or string literals.

But, my God is that a horrible configuration string. I mean, you might as well not have one considering nobody is likely to touch that. :P